src/com/sun/ts/tests/jpa/core/annotations/mapkeytemporal/EmbeddedEmployee.java

```diff
@@ -27,13 +27,13 @@
   private static final long serialVersionUID = 22L;
   public String employeeName;
   public int employeeId;
-
-  public EmbeddedEmployee() {
-    // for JPA
-  }
-
-  public EmbeddedEmployee(Employee4 employee) {
-    this.employeeName = employee.getLastName();
-    this.employeeId = employee.getId();
-  }
-}
\ No newline at end of file
+
```

Comment:
The changes on this file, don't look needed.

Reply:
This was caused by sed utility which replaced CRLF with LF. Will revert back.

Reply:
Fixed.